core/src/components.d.ts

```diff
@@ -1,9 +1,8 @@
-/* tslint:disable */
 /**
- * This is an autogenerated file created by the Stencil compiler.
- * It contains typing information for all components that exist in this project.
- */
-
+* This is an autogenerated file created by the Stencil compiler.
+* It contains typing information for all components that exist in this project.
+*/
+/* tslint:disable */
```

Comment:
Can you pull master, and npm i? looks like it's using an older version of stencil